ecom_test/src/generic_helpers/generic_order_helpers.py

```diff
@@ -0,0 +1,112 @@
+from ecom_test.src.dao.order_dao import OrderDAO
+from ecom_test.src.dao.products_dao import ProductsDAO
+from ecom_test.src.api_helpers.OrdersAPIHelper import OrdersAPIHelper
+
+import json
+import os
+
+
+class GenericOrderHelpers:
+
+    def __init__(self):
+        # Initialize instances of helper classes
+        self.product_dao = ProductsDAO()  # Helps retrieve product information from the database
+        self.order_api_helper = OrdersAPIHelper()  # Assists in making API calls related to orders
+        self.cur_file_dir = os.path.dirname(os.path.realpath(__file__))  # Gets the directory path of this script
+
+    def create_order(self, additional_args=None, expected_status_code=201):
+        """
+        Creates an order with provided or default payload, and returns the API response.
+
+        Args:
+        Args:
+            additional_args (dict, optional): Additional data to customize the order payload.
+            expected_status_code (int, optional): The expected HTTP status code for the API response.
+
+           Coupon: to add coupon add "coupon_lines" field to "additional_args"
```

Comment:
This looks like 'Coupon' is another parameter. And move it up so it is under additiona_args and specify "Example".

Reply:
resolved